docs/source/user_guides/basic_conceptions.rst

```diff
@@ -97,16 +111,8 @@ Let's start from creating the very simple dialog agent:
         fallback_label=("ping_pong_flow", "fallback_node"),
     )
 
-    if __name__ == "__main__":
-        pipeline.run()
```

Comment:
Why remove this block? Now it is unclear how to use pipeline after you've written the script.